main.cpp

```diff
@@ -73,12 +23,26 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void ToString() {
+    std::cout<<"REFERENCE MANUAL OF NUMBER TO COLOR MAPPING"<<std::endl;
```

Comment:
ToString is supposed to make a big string with the reference manual in it. It is not sufficient if it just outputs on console...